image/templates/helm/stackrox-central/templates/_init.tpl.htpl

```diff
@@ -98,14 +98,14 @@
 {{ include "srox.autoSenseOpenshiftVersion" (list $) }}
 
 {{/* Openshift monitoring */}}
-{{ if $._rox.enableOpenShiftMonitoring }}
+{{ if $._rox.monitoring.openshift.enabled }}
   {{ if (ne $._rox.env.openshift 4) }}
-    {{ include "srox.fail" "'enableOpenShiftMonitoring' is set to true, but the chart is not being deployed in an OpenShift 4 cluster." }}
+    {{ include "srox.fail" "'monitoring.openshift.enabled' is set to true, but the chart is not being deployed in an OpenShift 4 cluster." }}
   {{ end }}
 
   {{/* Override monitoring exposure when openshift monitoring is desired */}}
   {{ if not $._rox.central.exposeMonitoring }}
-    {{ include "srox.warn" (list . "The default or set value of 'central.exposeMonitoring' is false but 'enableOpenShiftMonitoring' is true.") }}
+    {{ include "srox.warn" (list . "The default or set value of 'central.exposeMonitoring' is false but 'monitoring.openshift.enabled' is true.") }}
```

Comment:
See my other comment, I guess it's fine to leave it like this for this PR, but this needs to be changed afterwards such that both flags can be set independently.

Reply:
Yup, I'll now focus on having it shaped as we like and then adjusting logic when we have endpoint in place.